plugins/org.jboss.tools.central.reddeer/META-INF/MANIFEST.MF

```diff
@@ -24,6 +24,7 @@ Bundle-ActivationPolicy: lazy
 Bundle-RequiredExecutionEnvironment: JavaSE-1.6
```

Comment:
If this is a problem, we should address it on whole repository level as it is in a lot of our plugins: http://i.imgur.com/dK3qbfx.png